torch/nn/quantized/functional.py

```diff
@@ -362,6 +362,27 @@ def max_pool2d(input, kernel_size, stride=None, padding=0, dilation=1,
     return torch.nn.functional.max_pool2d(input, kernel_size, stride, padding,
                                           dilation, ceil_mode, return_indices)
 
+def celu(input, alpha=1., inplace=False):
+    # type: (Tensor, Optional[float], Optional[bool]) -> Tensor
+    r"""celu(input, inplace=False) -> Tensor
+
+    Applies the quantized continuously differentiable ELU function element-wise.
```

Comment:
ELU -> CELU

Reply:
CELU stands for continuously differentiable ELU -- maybe I should capitalize the C to make it clearer this is what I mean?